gossip/src/crds_value.rs

```diff
@@ -84,8 +84,8 @@ pub enum CrdsData {
     ContactInfo(ContactInfo),
     Vote(VoteIndex, Vote),
     LowestSlot(/*DEPRECATED:*/ u8, LowestSlot),
-    SnapshotHashes(SnapshotHash),
-    AccountsHashes(SnapshotHash),
+    SnapshotHashes(SnapshotHashes),
+    AccountsHashes(SnapshotHashes),
```

Comment:
In this PR comment (https://github.com/solana-labs/solana/pull/20374#discussion_r721400651), Behzad mentioned that following the naming scheme of (old) `SnapshotHashes` wasn't the right call. So here's the update to make this better.